### PR TITLE
Fixed bounty button & added bounty to all

### LINKED ISSUE #2592: [Bug]: Set Bounty does not work
### Describe the bug

Setting a bounty on any player through YimMenu, does not work.

### Steps To Reproduce

1. Select any player
2. Put any amount for the bounty
3. Press "Set" with or without Anonymous Bounty
4. Will not work

### Expected Behavior

I expected it to work.

### Logs and/or screenshots

_No response_

### Store

Epic Games

### Language

English

### YimMenu Language

English (US)

### Additional context

This feature has not worked since version 2944. I thought it could have been my end but I have tried with every protection disabled and I also tried this on unmodded clients.

### Build

- [X] I've confirmed the issue exists on the latest version of YimMenu

### Coload

- [X] I've tested this without co-loading anything.

### PR DESCRIPTION
**Pull request by Tayipc38**
This took me longer than I'd like to admit. The issue was 2 imgui elements sharing the same ID.

Fixes #2592

### CODE REVIEW

File: src/util/troll.hpp [hunk 1]
```diff
@@ -36,5 +36,4 @@ namespace big::troll
 
 		g_pointers->m_gta.m_trigger_script_event(1, args, arg_count, -1, (int)eRemoteEvent::Bounty);
 	}
-
```
Comment (gir489returns) [inline src/util/troll.hpp:39]: Unnecessary whitespace change.

**Review by gir489returns — APPROVED** [review]
Looks good to me. There's some whitespace change in troll.hpp for no reason, but not even worth making another commit to fix.

File: src/util/troll.hpp (hunk 1 above)
Comment (Tayipc38) [reply src/util/troll.hpp:39]: The whitespace itself was unnecessary too. Ive developed minor ocd.

Comment (BigboiKilljoy) [reply src/util/troll.hpp:39]: You guys all hard at work making this menu very cool. Nice casino hack btw gir

**Review by Yimura — APPROVED** [review]

**Pull request merged**